src/containers/BrowseApps/index.jsx

```diff
@@ -5,15 +5,18 @@ import Nav from 'components/Nav';
 import AppList from 'components/AppList';
 import styles from './index.scss';
 
-@inject('rootStore')
+@inject(({ rootStore }) => ({
```

Comment:
i  think no need to change original logic, just inject `rootStore`, it's simple

Reply:
just pass what the component need